Xamarin.Forms.Platform.iOS/Renderers/SliderRenderer.cs

```diff
@@ -18,7 +21,9 @@ public override SizeF SizeThatFits(SizeF size)
 		protected override void Dispose(bool disposing)
 		{
 			if (Control != null)
+			{
 				Control.ValueChanged -= OnControlValueChanged;
+			}
```

Comment:
Here if the _sliderTapRecognizer is not null can you remove it from the control? That will probably fix the NRE exception I commented about as well but probably good to just remove it anyway to avoid other side effects.

Reply:
@PureWeen I've fixed this now - great you noticed this! Don't think this was the cause to the NRE though.